cmd/kubelet/app/server.go

```diff
@@ -95,6 +96,7 @@ type KubeletServer struct {
 	TLSPrivateKeyFile              string
 	CertDirectory                  string
 	NodeStatusUpdateFrequency      time.Duration
+	ChaosChance                    float64
```

Comment:
These are internal configs, so it's appropriate to put them under the struct.  Globals are far worse and break encapsulation so I don't think we should ever add those (ReallyCrash is probably the exception).  

I don't see a ton of value in separating them out, although I'd be happy to add a better description of when you would use those and separate them visually.

----- Original Message -----
> > @@ -95,6 +96,7 @@ type KubeletServer struct {
> >  	TLSPrivateKeyFile              string
> >  	CertDirectory                  string
> >  	NodeStatusUpdateFrequency      time.Duration
> > +	ChaosChance                    float64
> 
> Not a fan of having oddball testing options in this struct
> ("ReallyCrashForTesting"?). I'd prefer is these settings were coalesced into
> a separate struct and referenced either by a pointer from KubeletServer or
> perhaps just as a floating global config (presumably there won't be more
> than 1 per process...).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6729/files#r28259029